content/release/ReleaseNotes346.md

```diff
@@ -0,0 +1,52 @@
+<!--
+title: "Contrast 3.4.6 - November 2017"
+description: "Contrast 3.4.6 November 2017"
+tags: "3.4.6 November Release Notes"
+-->
+
+Tame unruly applications with improved Assess Rules, unique display names and a new Protect rule for Node.js. 
+
+## Fixes
+
+* Use date, build number and server filters to find the right vulnerabilities in the grid. 
+* Assess Rules can only be edited by admins, but anyone can filter by language. 
+* Security Settings are sorted out. 
+* Create Virtual Patches from Attack Events. 
+* Tuned up JIRA configuration fields so you can use default values for required fields, add assignees and update issue types without any problems. 
+* Use time filters to find Attack Events. 
+* Save changes in Password Settings without a confusing confirmation dialog. 
+* No more errors in server statistics.
+* Add users to an application access group during bulk import. 
+* View vulnerability event details more clearly. 
+* Get only relevant info from Organization Statistics and Attacks pages. 
+* Cleanly delete properties from vulnerabilities. 
+ 
+## Improvements 
+
+* Ever wondered if you could apply different [Assess Rules](admin-policymgmt.html#assess) to different applications? What if you want to track SQLi vulnerabilities on one application but not on another? With Assess Rules by application, your dreams for Java, Node and Ruby can come true. (And .NET will be coming soon!)
+
```

Comment:
I think only Ruby has implemented this feature. I know that Java and .NET haven't. It also looks like Node hasn't had a chance to yet based on this epic: CONTRAST-17698

Reply:
Removing this item completely until it's ready for all Assess agents per Product's recommendation.